### ISSUE TITLE
Boulders on Level 8 do not cause damage

### ISSUE THREAD

**Issue by TheBigStinkyBee** [jorio/Bugdom#36]
Just as the title suggests, the tumbling boulders on Level 8 don't hurt you or kill enemies when they fall down the hill. They just push you aside.

**Comment by jorio** [comment]
The boulders are supposed to deal some damage, but I just found out that whether or not they actually do is framerate-dependent. The higher the framerate, the less likely it is that the boulders cause damage.

@TheBigStinkyBee What's your screen's refresh rate? You can press F8 in-game to bring up the FPS counter.

**Comment by TheBigStinkyBee** [comment]
144Hz. The game is running at 144 fps. Is there a way to reduce it?

**Comment by TheBigStinkyBee** [comment]
Never mind, I changed it in my Display settings. It worked! Still, it seems a bit odd that sort of thing is framerate-dependent.

**Comment by jorio** [comment]
Yes, this is definitely a gameplay bug. I fixed the most glaring framerate-dependent (FRD) bugs in the game - the physics in slopes, platforms, fences, etc. weren't very tolerant of arbitrary framerates - but some subtle issues remain. I just did some testing at high framerates and I identified these problems:

- lv3: fish skedaddles offscreen at lightning speed after gobbling up Rollie
- lv4: giant feet on steroids
- lv5: once the hive is destroyed, the fire effect won't start if the framerate exceeds 100 fps
- lv7: Rollie's speed is FRD when trapped in a wad of honey
- lv8: boulder damage condition is FRD

I intend to fix these for the next release. Let me know if you find more so I can try to fix them.
